editor/src/components/navigator/navigator-conditionals.spec.browser2.tsx

```diff
@@ -173,6 +182,66 @@ export var ${BakedInStoryboardVariableName} = (
 `
 }
 
+function getProjectCodeEmptyActive(): string {
+  return `import * as React from 'react'
+import { Scene, Storyboard } from 'utopia-api'
+
+export var ${BakedInStoryboardVariableName} = (
+  <Storyboard data-uid='${BakedInStoryboardUID}'>
+    <Scene
+      style={{
+        backgroundColor: 'white',
+        position: 'absolute',
+        left: 0,
+        top: 0,
+        width: 400,
+        height: 700,
+      }}
+      data-uid='${TestSceneUID}'
+      data-testid='${TestSceneUID}'
+    >
+      <div
+        style={{
+          height: '100%',
+          width: '100%',
+          contain: 'layout',
+        }}
+        data-uid='containing-div'
+        data-testid='containing-div'
+      >
+        {[].length === 0 ? (
+          [].length === 0 ? null : null
+        ) : (
+          <div
+            style={{
+              height: 150,
+              position: 'absolute',
+              left: 154,
+              top: 134,
+            }}
+            data-uid='else-div'
```

Comment:
Can you rename the uses of "then" and "else" to match up with #3428 please

Reply:
Seems pretty inconsequential given this is test data surely?